SPECS/curl/curl.spec

```diff
@@ -87,6 +87,8 @@ rm -rf %{buildroot}/*
 %{_libdir}/libcurl.so.*
 
 %changelog
+*   Wed Oct 07 2020 Pawel Winogrodzki <joschmit@microsoft.com> 7.68.0-2
```

Comment:
wrong email. Also, there should be a `-` before the version

Reply:
Same for other SPECs

Reply:
Good catch, thanks!